### PR TITLE
Nonprofit criteria2

### PR DESCRIPTION
**Pull request by jfernan123**

### CODE REVIEW

**Comment by dimassudjito** [comment]
Code Review Checklist

General Check

- [x]  Check that the code compiles and runs
- [x]  Check that the code is producing the required output by comparing it to the acceptance criteria in the GitHub issues. (e.g., compare it with Figma sketch for frontend work)
- [x]  Check that the code has acceptable tests and that the tests are passing.
- [x]  Check that the PR is directed at the correct branch (develop Vs. main)
- [x]  Check that the PR does not have merge conflict
- [x]  Check that CI/CD tests are passing

Implementation Review

 

- [x] Understand implementation approach and give feedback on how we can improve the general approach
- [x]  Understand data structures being used in the implementation and provide feedback (i.e., is it optimal?)
- [x]  Understand external libraries used in the implementation and provide feedback (i.e., is it deprecated? Is there better libraries out there?)
- [x]  Check for repeated code and give feedback
- [x]  Check for performance-intensive code (e.g., heavy algorithm) and give feedback
- [x]  Check for dead code or commented code that should be deleted
- [x]  Check if variable names are sensible and descriptive

File: frontend/src/tests/nonprofitApply/nonprofitApplyContent.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```
Comment (dimassudjito) [inline frontend/src/tests/nonprofitApply/nonprofitApplyContent.test.js]: (nit) can change name to criteriaContent.test.js

File: frontend/src/tests/nonprofitApply/nonprofitApplyImage.test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```
Comment (dimassudjito) [inline frontend/src/tests/nonprofitApply/nonprofitApplyImage.test.js]: (nit) can change name to criteriaImage.test.js

File: frontend/src/tests/nonprofitApply/nonprofitApplyTitle.test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```
Comment (dimassudjito) [inline frontend/src/tests/nonprofitApply/nonprofitApplyTitle.test.js]: (nit) can change name to criteriaTitle.test.js

**Review by dimassudjito — CHANGES_REQUESTED** [review]
This looks good to me. Just a minor problem in the test with naming and location; I think the tests are failing because you accidentally put the nonprofitApply test folder in tests/ instead of tests/components (it results in import error)

File: frontend/src/tests/nonprofitApply/nonprofitApplyTitle.test.js (hunk 3 above)
Comment (jfernan123) [reply frontend/src/tests/nonprofitApply/nonprofitApplyTitle.test.js]: done

File: frontend/src/tests/nonprofitApply/nonprofitApplyImage.test.js (hunk 2 above)
Comment (jfernan123) [reply frontend/src/tests/nonprofitApply/nonprofitApplyImage.test.js]: done

File: frontend/src/tests/nonprofitApply/nonprofitApplyContent.test.js (hunk 1 above)
Comment (jfernan123) [reply frontend/src/tests/nonprofitApply/nonprofitApplyContent.test.js]: done

**Review by dimassudjito — APPROVED** [review]

**Pull request merged**